content/technotes/2020-02-26-tic-ghactions.Rmd

```diff
@@ -50,42 +50,86 @@ Furthermore {tic} aims to simplify the tedious process of setting up deployment
 ("Deployment" means that a CI run is allowed to perform a `git` push to the repository.)
 This is achieved by its helper packages such as {circle}, {ghactions} or {travis} which perform API calls to the respective CI providers.
 
+## GitHub Actions - the new kid on the block
 
+GitHub Actions is around since about one year.
```

Comment:
Not really in its current version? See e.g. https://siliconangle.com/2019/11/13/github-universe-announcements-bring-bevy-updates-developers/

Reply:
Ok then let's refer to the official release date :)